core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -184,7 +184,7 @@ ${h.initPageVariables(context)}
       <div id="header">
         <div class="logo">
           <a id="jenkins-home-link" href="${rootURL}/">
-            <img id="jenkins-head-icon" src="${imagesURL}/headshot.png" alt="title" />
+            <img id="jenkins-head-icon" src="${imagesURL}/headshot.png" alt="[${%jenkinshead.alt}]" />
```

Comment:
Unclear why `[]`

Reply:
Brackets are used to indicate something like "this is a picture of an x" as opposed to "this is the text that the image x presented".

See http://jkorpela.fi/html/alt.html#thumb point 8